src/orquestra/sdk/_base/_driver/_client.py

```diff
@@ -694,7 +692,9 @@ def get_task_run_logs(self, task_run_id: _models.TaskRunID) -> bytes:
         # TODO: unzip, get logs (ORQSDK-654)
         return resp.content
 
-    def get_system_logs(self, wf_run_id: _models.WorkflowRunID) -> List[str]:
+    def _get_system_logs(
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏Is there a reason why this is `_` prefixed?

Reply:
There was, but it's now outdated. I've changed it back to non `_` - prefixed